oidc-client.js

```diff
@@ -294,6 +296,10 @@ OidcClient.prototype.createLogoutRequestAsync = function (id_token_hint) {
         if (id_token_hint && settings.post_logout_redirect_uri) {
             url += "?post_logout_redirect_uri=" + encodeURIComponent(settings.post_logout_redirect_uri);
             url += "&id_token_hint=" + encodeURIComponent(id_token_hint);
+
+            if (state !== undefined) {
+                url += "&state=" + encodeURIComponent(state);
+            }
```

Comment:
Just one comment about this -- not sure what you're passing for state, but if it's structured data in any way (other than just a nonce type of value) then there's a possibility of it being tampered with before you return back to your app. I'd not trust it for anything sensitive.